src/logger/index.ts

```diff
@@ -13,15 +13,10 @@ const levels: { [key: number]: string } = {
 // Type for log objects
 type LogObject = {
   level?: number | string;
-  // TODO [explicit-any]: consider add type if it's possible
-  // eslint-disable-next-line  @typescript-eslint/no-explicit-any
-  [key: string]: any;
+  [key: string]: unknown;
```

Comment:
Do we really don't know what value we could have here?

Reply:
Almost everything can be here. From undefined to custom types.